core/src/main/resources/jenkins/install/pluginSetupWizard.properties

```diff
@@ -56,3 +58,17 @@ installWizard_pluginInstallFailure_message=Some plugins failed to install proper
 with the failed plugins
 installWizard_continue=Continue
 installWizard_retry=Retry
+installWizard_upgradePanel_title=Upgrade
+installWizard_upgradePanel_banner=Welcome to Jenkins 2!
+installWizard_upgradePanel_message=Jenkins 2 includes some great new features we think you'll love to use. Get your instance updated to the latest \
+and greatest plugins available and take advantage of them!
+installWizard_upgradePanel_recommendedActionTitle=Install Recommended Plugins
+installWizard_upgradePanel_recommendedActionDetails=Install the recommended plugins
+installWizard_upgradePanel_skipRecommendedPlugins=No thanks
+installWizard_upgradeComplete_title=Upgrade
+installWizard_pluginsInstalled_banner=Welcome to Jenkins 2!
+installWizard_upgradeComplete_message=Congratulations! You've upgraded to Jenkins 2.</p><p>To learn more about the great new features, visit <a target="_blank" href="https://jenkins.io/2.0/">the Jenkins 2 website</a>!
```

Comment:
> You've upgraded

Better just to write "You have upgraded", some people get weird about the contraction "you've"